src/test/java/org/cloudfoundry/promregator/config/validations/PreferredRouteRegexMustBeCompilableTest.kt

```diff
@@ -0,0 +1,25 @@
+package org.cloudfoundry.promregator.config.validations
+
+import org.cloudfoundry.promregator.lite.config.PromregatorConfiguration
+import org.cloudfoundry.promregator.lite.config.Target
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.Test
+
+internal class PreferredRouteRegexMustBeCompilableTest {
+
+    @Test
+    fun testValidDoesNotBreak() {
+        val subject = PreferredRouteRegexMustBeCompilable()
+        val target = Target(preferredRouteRegex = listOf("dummy"))
```

Comment:
Migrated this test to Kotlin so the simplified named parameter constructor could be used.